hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/HdfsNode.java

```diff
@@ -210,12 +209,17 @@ protected boolean enoughResources(Offer offer, double cpus, int mem) {
   private List<Task> createTasks(Offer offer) {
     String executorName = getExecutorName();
     String taskIdName = String.format("%s.%s.%d", name, executorName, System.currentTimeMillis());
-    List<Task> tasks = new ArrayList<Task>();
```

Comment:
I do agree that simpler is better. The problem is that we have style inconsistency since in other places we do the full declaration. I suppose this is fine, and we can update the other declarations to use this simpler style as we go.